packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -220,4 +221,45 @@ describe('Dialog', () => {
       expect(submitMessage.args).toEqual(['someResult', ['someAppId']]);
     });
   });
+  describe('sendMessageToParentFromDialog', () => {
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const emptyCallback = () => {};
+    const allowedContexts = [FrameContexts.task];
+    it('should not allow calls before initialization', () => {
+      expect.assertions(1);
+      expect(() => dialog.sendMessageToParentFromDialog('message')).toThrowError(
+        'The library has not yet been initialized',
+      );
+    });
+
+    Object.keys(FrameContexts)
+      .map(k => FrameContexts[k])
+      .forEach(frameContext => {
+        if (frameContext === FrameContexts.task) {
+          it(`should successfully send the message to Parent: ${frameContext}`, async () => {
+            await utils.initializeWithContext(frameContext);
+            utils.setRuntimeConfig({ apiVersion: 1, supports: { dialog: {} } });
+            dialog.sendMessageToParentFromDialog('exampleMessage', emptyCallback);
+            const message = utils.findMessageByFunc('messageForParent');
+            expect(message).not.toBeUndefined();
+          });
+        } else {
+          it(`should not allow calls from ${frameContext} context`, async () => {
+            await utils.initializeWithContext(frameContext);
+            expect(() => dialog.sendMessageToParentFromDialog('message', emptyCallback)).toThrowError(
+              `This call is only allowed in following contexts: ${JSON.stringify(
+                allowedContexts,
+              )}. Current context: "${frameContext}".`,
+            );
+          });
+        }
+      });
+  });
+  it('registerOnMessageFromParent', () => {
```

Comment:
this should probably be in its own describe block, and the test itself should have a more descriptive name like the others you wrote above.